src/backend/events/filters/builtin-filter-loader.ts

```diff
@@ -1,8 +1,11 @@
-"use strict";
+import filterManager from "./filter-manager";
+import metadataKeyFilter from "./builtin/metadata-key";
+import metadataValueFilter from "./builtin/metadata-value";
 
-const filterManager = require("./filter-manager");
+export function loadFilters() {
+    filterManager.registerFilter(metadataKeyFilter);
```

Comment:
In terms of consistency, I'm not sure if this is the right way to add back these filters. The effects deal with a similar mix of js and ts, so possibly it's good to have a look at how those are handled and apply the same solution here (for now).